.github/workflows/translation_status.yml

```diff
@@ -19,6 +19,26 @@ jobs:
       
       - name: dbt dependencies
         run: "dbt deps"
+
+      - name: Run dbt debug on Trino if dunesql engine is used, and retry until cluster is available
+        if: matrix.engine == 'dunesql'
+          run: |
+            RETRY_COUNT=0
+            # max wait: 10 minutes
+            MAX_RETRIES=40
+            WAIT_TIME=15
+            
+            until [ "$RETRY_COUNT" -ge "$MAX_RETRIES" ]
+            do
+              dbt debug $PROFILE && break
+              RETRY_COUNT=$((RETRY_COUNT+1))
+              if [ "$RETRY_COUNT" -ge "$MAX_RETRIES" ]; then
+                echo "Max retries reached, failing..."
+                exit 1
+              fi
+              echo "Retrying in $WAIT_TIME seconds... ($RETRY_COUNT/$MAX_RETRIES)"
+              sleep $WAIT_TIME
+            done
```

Comment:
Can we extract this in a script so that we don't repeat it three times in the different workflow files? 🙏

Reply:
Sure thing 👍 should the script live on spellbook or in the infra repo where we define GH runners logic?

Reply:
Let's have it here alongside the actions 👍